net/discovery.go

```diff
@@ -10,9 +10,9 @@ import (
 	libp2phost "github.com/libp2p/go-libp2p-core/host"
 	"github.com/libp2p/go-libp2p-core/peer"
 	"github.com/libp2p/go-libp2p-core/peerstore"
-	libp2pdiscovery "github.com/libp2p/go-libp2p-discovery"
```

Comment:
This is what prevented the upgrade from being trivial. The repo is deprecated and the code was moved into packages of `github.com/libp2p/go-libp2p`:  https://github.com/libp2p/go-libp2p-discovery/commit/111203389c4886581007b0837ba9d48821f4fb6f